notify/notification_apns.go

```diff
@@ -406,7 +414,7 @@ func PushToIOS(req *PushNotification, cfg *config.ConfYaml) (resp *ResponsePush,
 Retry:
 	var newTokens []string
 
-	notification := GetIOSNotification(req)
+	notification, _ := GetIOSNotification(req)
```

Comment:
You can't ignore `err` here. In case of `err != nil`, `notification = nil` and we have panic later.

Reply:
Hi @slimus , just because **log** parser error, we don't have to return err parameter, so I removed it from return statement